tests/hw8/test_task1.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+
+from hw8.task1 import KeyValueStorage
+
+
+@pytest.fixture(scope="module")
+def key_value_storage_instance():
+    instance = KeyValueStorage("task1.txt")
+    storage = instance._get_copy_of_storage_dict()
+    yield instance
+    for key, value in storage.items():
+        instance[key] = value
+
+
+def test_reading_char_and_integer_value_from_file(key_value_storage_instance):
```

Comment:
if we do in such way (separated functions instead of class), KeyValueStorage instance is being created and the file is readed for every test function - this is a bit overwhelmed. Probably a better idea is to make in in the test class and make kind of `def setup()` fixture inside of it

Reply:
Done